### ISSUE TITLE
Initial pitch review

### ISSUE THREAD

**Issue by tigpt** [global-aws-community/aws-ug-cookbook#9]
https://github.com/global-aws-community/aws-ug-cookbook/blob/main/README.md

<img width="870" alt="image" src="https://github.com/user-attachments/assets/43d1790f-afde-4ac2-a057-9228739f73ff">

I've reviewed the initial pitch to be more clear the project objectives and also separated the how to contribute to its own page:
https://github.com/global-aws-community/aws-ug-cookbook/blob/main/how-to-contribute.md

Please take a look @diego7marques and @Fabrimonteiro and provide feedback.

**Comment by diego7marques** [comment]
@tigpt, the pitch seems accurate and very well written to me (congrats), the only small thing I think we can add is a statement to make the target audience clear and avoid questions like: Is the repository for AWS UG Leaders? Is it for members? 

Trying to address that, I've opened this PR: https://github.com/global-aws-community/aws-ug-cookbook/pull/13

I like the way you sorted the "How to contribute" page into his own file. It also gave me another ideia: we should create issue templates to make easier for people to contribute giving them a starting point with a set of questions. I've created the issue: https://github.com/global-aws-community/aws-ug-cookbook/issues/14 (I did this separated so the issue templating do not block the pitch evolution promotion to main)
